### ISSUE TITLE
Trying to get property of non-object (class-wcmp-export.php on line 1470)

### ISSUE THREAD

**Issue by devos50** [myparcelnl/woocommerce#229]
I'm getting the following error when performing a payout in Woocommerce (version 3.1.4):

```
Trying to get property of non-object in /home/jbijl/domains/hengelsportoosterschelde.nl/public_html/wp-content/plugins/woocommerce-myparcel/includes/class-wcmp-export.php on line 1470
```

Is this a known issue and is there any quick workaround for this?

**Comment by adem-demir** [comment]
Dear Devos50,

Thanks for your feedback. Can you please add **(int)** before **$product->get_weight();** on row **1258**

We will add this in the next version. Please let me know if it doesn't works.

**Comment by RichardPerdaan** [comment]
This issue has been resolved since [3.1.5](https://github.com/myparcelnl/woocommerce/releases/tag/v3.1.5)

**Comment by devos50** [comment]
Thanks for fixing this issue! I already fixed it by another workaround but wil upgrade as soon as I can.
